databricks_cli/unity_catalog/metastore_cli.py

```diff
@@ -193,6 +206,8 @@ def register_metastore_commands(cmd_group):
     cmd_group.add_command(hide(metastore_summary_cli), name='metastore-summary')
     cmd_group.add_command(hide(assign_metastore_cli), name='assign-metastore')
     cmd_group.add_command(hide(unassign_metastore_cli), name='unassign-metastore')
+    cmd_group.add_command(hide(get_metastore_assignment_cli),
```

Comment:
Do we need this? Based on the comment, these are "deprecated" commands

Reply:
No, I just kinda liked the `get-metastore-assignment` version.  Can remove it, if you prefer.